Documentation/mkfs.btrfs.rst

```diff
@@ -160,6 +160,10 @@ OPTIONS
         directory.  The option *--rootdir* must also be specified, and *subdir* must be an
         existing subdirectory within it.  This option can be specified multiple times.
 
+-D|--default-subvol <subdir>
```

Comment:
Not sure if we want a short option for this.
Especially this option is dependent on both `--rootdir` and `--subvolume`